packages/cozy-harvest-lib/src/helpers/queries.js

```diff
@@ -28,10 +28,13 @@ export const buildAppsRegistryQueryBySlug = slug => {
   return { definition: query.definition(), options: query.options }
 }
 
-export const buildAccountQuery = accountId => ({
-  definition: Q('io.cozy.accounts').getById(accountId),
+export const buildAccountQuery = ({ slug, sourceAccountIdentifier }) => ({
+  definition: Q('io.cozy.accounts').where({
+    account_type: slug,
+    'cozyMetadata.sourceAccountIdentifier': sourceAccountIdentifier
+  }),
   options: {
-    as: `io.cozy.accounts/${accountId}`,
+    as: `account_type/${slug}/io.cozy.accounts.sourceAccountIdentifier/${sourceAccountIdentifier}`,
```

Comment:
Je te renvoi à la guideline [naming-of-queries](https://github.com/cozy/cozy-guidelines?tab=readme-ov-file#naming-of-queries).
Donc plutôt quelque chose du genre je pense :
```
io.cozy.accounts/slug/${slug}/sourceAccountIdentifier/${sourceAccountIdentifier}
```

Reply:
Ok, mais j'ai quand même gardé account_type au lieu de slug puisque que c'est bien l'attribut account_type qu'on cherche.